moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -275,309 +277,537 @@ const ProjectsListViewTable = ({ query, searchTerm }) => {
           state={{
             filters: Object.keys(filters).length
               ? btoa(JSON.stringify(filters))
               : false,
           }}
           className={classes.colorPrimary}
         >
-          {entry.project_name}
+          {entry.value}
         </RouterLink>
       ),
-      cellStyle: {
-        position: "sticky",
-        left: 0,
-        backgroundColor: "white",
-        minWidth: "20rem",
-        zIndex: 1,
-      },
+      width: 330,
     },
     {
-      title: "Status",
+      headerName: "Status",
       field: "current_phase",
-      hidden: hiddenColumns["current_phase"],
-      render: (entry) =>
+      renderCell: (entry) =>
         buildStatusBadge({
-          phaseName: entry.current_phase,
-          phaseKey: entry.current_phase_key,
+          phaseName: entry.row.current_phase,
+          phaseKey: entry.row.current_phase_key,
         }),
+      width: 210,
     },
     {
-      title: "Team",
+      headerName: "Team",
       field: "project_team_members",
-      hidden: hiddenColumns["project_team_members"],
-      cellStyle: { whiteSpace: "noWrap" },
-      render: (entry) =>
+      renderCell: (entry) =>
         renderProjectTeamMembers(
-          entry.project_team_members,
+          entry.row.project_team_members,
           "projectsListView"
         ),
+      width: 350,
     },
     {
-      title: "Lead",
+      headerName: "Lead",
       field: "project_lead",
-      hidden: hiddenColumns["project_lead"],
-      editable: "never",
-      cellStyle: { whiteSpace: "noWrap" },
-      render: (entry) =>
-        entry.project_lead === null ? "-" : entry.project_lead,
+      renderCell: (entry) =>
+        entry.row.project_lead === null ? "-" : entry.row.project_lead,
+      width: 200,
     },
     {
-      title: "Sponsor",
+      headerName: "Sponsor",
       field: "project_sponsor",
-      hidden: hiddenColumns["project_sponsor"],
-      editable: "never",
       cellStyle: { whiteSpace: "noWrap" },
-      render: (entry) =>
-        entry.project_sponsor === "None" ? "-" : entry.project_sponsor,
+      renderCell: (entry) =>
+        entry.row.project_sponsor === "None" ? "-" : entry.row.project_sponsor,
+      width: 200,
     },
     {
-      title: "Partners",
+      headerName: "Partners",
       field: "project_partner",
-      hidden: hiddenColumns["project_partner"],
-      emptyValue: "-",
-      cellStyle: { whiteSpace: "noWrap" },
-      render: (entry) => {
-        return entry.project_partner.split(",").map((partner) => (
-          <span key={partner} style={{ display: "block" }}>
-            {partner}
-          </span>
-        ));
+      renderCell: (entry) => {
+        return entry.row.project_partner
+          ? entry.row.project_partner.split(",").map((partner) => (
+              <span key={partner} style={{ display: "block" }}>
+                {partner}
+              </span>
+            ))
+          : "-";
```

Comment:
material table has a property called "emptyValue" that we defined, but DataGrid does not